classes/emailTemplate/DAO.inc.php

```diff
@@ -399,4 +399,34 @@ public function installEmailTemplateData(
         }
         return true;
     }
+
+    /**
+     * @param string $localizedData email template's localized subject or body
+     * @return string
+     */
+    protected function renameVariables(string $localizedData): string
```

Comment:
Since this is primarily about the appliction/context, let's call this `renameApplicationVariables()`.